content/docs/guides/neon-authorize-clerk.md

```diff
@@ -39,89 +38,98 @@ In this first set of steps, we’ll integrate Clerk as an authorization provider
 
-![Add Authentication Provider](/docs/guides/clerk_jwks_url_in_neon.png)
+<div style={{ display: 'flex', justifyContent: 'center'}}>
+  <img src="/docs/guides/clerk_jwks_url_in_neon.png" alt="Add Authentication Provider" style={{ width: '60%', maxWidth: '600px', height: 'auto' }} />
+</div>
 
 At this point, you can use the **Get Started** setup steps from the Authorize page in Neon to complete the setup — this guide is modelled on those steps. Or feel free to keep following along in this guide, where we'll give you a bit more context.
 
 ### 3. Install the pg_session_jwt extension in your database
 
 Neon Authorize uses the [pg_session_jwt](https://github.com/neondatabase/pg_session_jwt) extension to handle authenticated sessions through JSON Web Tokens (JWTs). This extension allows secure transmission of authentication data from your application to Postgres, where you can enforce Row-Level Security (RLS) policies based on the user's identity.
 
-When installed, the `pg_session_jwt` enables passwordless connections by using JWTs for user authentication. Two roles are created for you: `authenticated` and `anonymous`.
-
 To install the extension in the `neondb` database, run:
 
-```sql
+```sql shouldWrap
 CREATE EXTENSION IF NOT EXISTS pg_session_jwt;
 ```
 
-<Admonition type="note">
-In a future update, setting up the `pg_session_jwt` extension and granting role privileges will be done automatically when you add an authentication provider to your Neon project.
-</Admonition>
-
-### 4. Set up Roles
+### 4. Set up Postgres roles
 
-Next, define the table-level permissions for these new roles. For most use cases of Neon Authorize, you should run the following commands in order to give the roles access to read and write to any table in your public schema:
+The integration creates the `authenticated` and `anonymous` roles for you. Let's define table-level permissions for these roles. To allow both roles to read and write to tables in your public schema, run:
 
 ```sql
 GRANT SELECT, UPDATE, INSERT, DELETE ON ALL TABLES IN SCHEMA public TO authenticated;
 GRANT SELECT, UPDATE, INSERT, DELETE ON ALL TABLES IN SCHEMA public TO anonymous;
 ```
 
-Later, you can define RLS policies that will restrict what your application's users can do with these roles.
+- **Authenticated role**: This role is intended for users who are logged in. Your application should send the authorization token when connecting using this role.
+- **Anonymous role**: This role is intended for users who are not logged in. It should allow limited access, such as reading public content (e.g., blog posts) without authentication.
 
 ### 5. Install the Neon Serverless Driver
 
 Neon’s Serverless Driver manages the connection between your application and the Neon Postgres database. It supports HTTP, WebSockets, and TCP connections. For Neon Authorize, the connection must be established over HTTP. For HTTP, we recommend using this driver.
```

Comment:
I wonder if we should avoid saying "recommend" unless there are other options. Are there other drivers that could be used?

Reply:
I can reword to make it clearer. In theory "the HTTP API can be used from any language and Authorize can be used with the HTTP API from any language" but we want to recommend using our driver.

`Neon’s Serverless Driver manages the connection between your application and the Neon Postgres database. For Neon Authorize, HTTP is requrired. While it is technically possible to access the HTTP API without using our driver, we recommend using the driver for best performance. The driver also supports WebSockets and TCP connections, so make sure you use the HTTP method when working with Neon Authorize.`